docs/installation/SUMMA_on_OS_X.md

```diff
@@ -54,12 +59,68 @@ In the following I will assume that you don't have a Fortran compiler or NetCDF
+
+ These instructions assume that you've already cloned the SUMMA repository.
+ If you haven't, follow the initial [SUMMA Installation Instructions](https://summa.readthedocs.io/en/latest/installation/SUMMA_installation/)
+
+### Step 1: Installation of Homebrew
+
+ 1. If you are already using MacPorts, it is recommended to uninstall it to avoid package issues.
+ Uninstallation instructions are available [here](https://guide.macports.org/chunked/installing.macports.uninstalling.html).
+ If you'd like to save the currently installed packages for MacPorts to file, use the command: `port installed > ports_installed.txt`
+
+ 2. Homebrew can then be installed following the instructions [here](https://brew.sh/)
+
+### Step 2: Installation of Required packages
+
+ 1. Use the command as follows to install gcc and gfortran compilers.
+ `brew install gcc`
+
+ 2. Use the command as follows to install netCDF.
+ `brew install netcdf`
+
+ Note that as a bonus, NetCDF Fortran is installed with this command and does not require action.
+ It is installed as a [resource](https://github.com/Homebrew/homebrew-core/blob/HEAD/Formula/netcdf.rb), by default.
+
+ 3. Use the following commands to install lapack and openblas using Homebrew
+ `brew install lapack`
+ `brew install openblas`
+
+### Step 3: Compile SUMMA
+  1. Now obtain the SUMMA source code from the [SUMMA source code repository](https://github.com/NCAR/summa). You may just want to download the latest tagged release. Unless you are planning to contribute to the source code, there is no need to clone or fork the repository.
+
+ 2. Untar or unzip the archive, then go to the `summa/build` directory and follow the instructions in the [SUMMA installation](SUMMA_installation.md) page. If you are using Homebrew, you can also follow the following steps:
+
+ 3. Navigate to your local copy of the SUMMA directory and go to the build subdirectory;
+ 4. Make a copy of the Makefile, naming it Makefile.local for your own use.
+ 5. Update the Environment Variables in the Makefile.local following the instructions. What worked for me is:
+
+ F_MASTER=$PATH_TO_SUMMA_GIT_CLONE$
+ FC=gfortran
+ FC_EXE=/usr/local/bin/gfortran
+ INCLUDES=-I/usr/local/include -I/usr/local/opt/lapack/include -I/usr/local/opt/openblas/include
+ LIBRARIES=-L/usr/local/lib -lnetcdff -L/usr/local/opt/lapack/lib -lblas -L/usr/local/opt/openblas/lib -lopenblas
+
+ While Homebrew uses the /usr/local/ dir and this should be consistent, it is recommend to check these folders to verify the files exist.
+
+ Navigating to the SUMMA base directory (i.e. F_MASTER, the root directory of the build folder), run the command
+
+ 6. `make -f build/Makefile.local`
+
+ If the code compiles successfully, then the last line of output from the make process will tell you where the SUMMA executable is installed (it goes into summa/bin). Run summa.exe in that directory (you may need to provide the full path).
```

Comment:
I asked @arbennett for a second opinion on this and he came up with what's possibly the best solution; merging the general and macOS-specific instructions onto a single page. The macOS instructions would simply be a sub-section on that page. That way there's less hunting around for info and given the similarity between compiling on linux and macOS, users of either system can benefit from seeing instructions for the other (and compiling on Windows isn't possible at all as far as I know, because there's no windows equivalent of the fortran-netcdf library). 

If you're up for it, can you merge this macOS page into the general instructions and tidy the whole thing up a bit?

If that's too much, than I think the general instructions would at least benefit from having a note added that points users to these instructions on the macOS page.